incubator/catalog-apicurio/src/main/zilla/internal.idl

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright 2021-2023 Aklivity Inc
+ *
+ * Licensed under the Aklivity Community License (the "License"); you may not use
+ * this file except in compliance with the License.  You may obtain a copy of the
+ * License at
+ *
+ *   https://www.aklivity.io/aklivity-community-license/
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OF ANY KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+scope internal
+{
+    option byteorder network;
+
+    struct ApicurioPrefix
+    {
+        uint8 magic = 0;
+        int32 schemaId;
+    }
+}
```

Comment:
Please confirm we are compatible with Apicurio for Kafka client serializers and deserializers.
https://www.apicur.io/registry/docs/apicurio-registry/2.5.x/getting-started/assembly-configuring-kafka-client-serdes.html

Looks like there is some variation available around long vs integer for id, and globalId vs contentId for resolving schemas.